capstone-project/src/main/webapp/map/map.component.js

```diff
@@ -88,7 +101,10 @@ angular.module('map').component('mapComponent', {
             });
         };
 
-        // Iterate over the markers json and add all of them to the map
-        $http.get('map/markers/markers.json').then((response) => angular.forEach(response.data, addMarkerForDisplay));
+        //Fetches markers from the backend and adds them to the map.
+        $http({
```

Comment:
Why not use the $http.get method? 

https://docs.angularjs.org/api/ng/service/$http#get